portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -55,19 +58,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     
     // Create the query and prepared query to load comment entities from database
     Query query = new Query(COMMENT).addSort(TIMESTAMP_PROPERTY, SortDirection.DESCENDING);
-
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-    
-    // Add limited number comment contents to the msglist  
-    ArrayList<String> msglist = new ArrayList<String>();
+
+    // Add limited number of comment contents and user email to the userComment Map  
+    HashMap<String, ArrayList<String>> userComment = new HashMap<String, ArrayList<String>>();
     for (Entity commentEntity:results.asList(FetchOptions.Builder.withLimit(maxNumComments))) {
```

Comment:
Missed this last time: space in between "commentEntity", ":" and "results":
"commentEntity : results..."